de.metas.async/src/main/java/de/metas/async/api/IAsyncBatchDAO.java

```diff
@@ -30,21 +32,62 @@
 
 import org.adempiere.util.ISingletonService;
 
+import de.metas.async.model.I_C_Async_Batch;
 import de.metas.async.model.I_C_Async_Batch_Type;
+import de.metas.async.model.I_C_Queue_WorkPackage;
+import de.metas.async.model.I_C_Queue_WorkPackage_Notified;
 
 /**
  * @author cg
  *
  */
 public interface IAsyncBatchDAO extends ISingletonService
 {
+	
+	public static final String ASYNC_BATCH_TYPE_INVCAND = "InvoiceCandidate";
```

Comment:
IMHO those IAsyncBatchDAO is a part of the infrastructure whereas ESR and InvoiceCands are concrete business applications. please move those constants to their respective modules.